### ISSUE TITLE
Upcoming changes

### ISSUE THREAD

**Issue by scj643** [Sinap/plugin.video.funimation#20]
Funimation is fixing stuff and this might break the plugin completely

More info http://www.funimationnow.com/

**Comment by Neogrey** [comment]
FunimationNow gets a cross-platform queue/history sync, would be a great feature for plugin .

**Comment by afrase** [comment]
Have they released any of the new apps yet?

**Comment by Neogrey** [comment]
US February:
- New device apps for Apple TV app, iOS, Android mobile and tablet, Windows 10 Universal app,   Amazon Kindle, and Amazon Fire TV.
- New update mobile apps will include Chromecast and Airplay support.

US May:
- Completely overhauled and new website with some added features and improved functionality that includes responsive design.
- New site will include Chromecast and Airplay support.
- App updates for PS3, PS4, Roku, Xbox One, and Xbox 360.

UK February:
- Website (Almost exactly like the one the US is getting in May, but a few features may be missing due to territory restrictions).
- Apps for iOS, Android mobile and tablet, Windows 10 Universal, Amazon Kindle, Amazon Fire TV, PS3, PS4, Roku, Xbox 360, and Xbox One.

You can contact funimation might help you: 

> As for help with the unofficial plugin, the people working on it would have to contact us. They can to through www.funimation.com/support and it will make its way towards me. They can choose "Other Technical". Then I can forward it on to the decision makers. Like with any app development pitch, the more information they give about what would be required and what having a plugin on xbmc could offer, the better.

**Comment by scj643** [comment]
They have now released the new apps
http://funimationnow.com/apps
The api is xml now and a real pain.

**Comment by afrase** [comment]
Yeah I see that. At least they are providing a lot more data and aren't doing any obfuscation.

**Comment by scj643** [comment]
But it also looks like they got their act together and now doing proper user auth.

**Comment by scj643** [comment]
Work should be started ASAP on the API since it will be unified.

**Comment by afrase** [comment]
I've already started working on a client https://github.com/Sinap/funimationlater

**Comment by Neogrey** [comment]
@Sinap  seems the last update is a few month ago (funimationlater) , no plans for a release of the plugin ?.
